Week2/homework/js-exercises/2-the-even-ood-reporter.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+for (var i = 0; i <= 20; i++) {
+    if (i % 2 === 0) {
+        console.log(i + ' is even');
+    } else {
+        console.log(i + ' is odd');
+    }
+}
```

Comment:
👍 but `let` instead of `var` please

Reply:
thank you so much Yash-kapila for the feedback I changed .